src/main/resources/META-INF/resources/primefaces/datepicker/1-datepicker.js

```diff
@@ -215,6 +215,23 @@ PrimeFaces.widget.DatePicker = PrimeFaces.widget.BaseWidget.extend({
         }
     },
 
+    /**
+     * Callback for after the overlay panel is created.
+     * @private
+     */
+    bindPanelCreationListener: function() {
+        var _self = this;
```

Comment:
I have a future PR where I will convert all _self to $this in the 1-datepicker.js as it doesn't do things like our other components and it should be consistent.